src/sentry/search/events/fields.py

```diff
@@ -2547,6 +2599,149 @@ def _resolve_unimplemented_alias(self, alias: str) -> SelectType:
         """
         raise NotImplementedError(f"{alias} not implemented in snql field parsing yet")
 
+    def _private_project_threshold_multi_function(self) -> SelectType:
```

Comment:
the single underscore already implies private
```suggestion
    def _project_threshold_multi_function(self) -> SelectType:
```

Also could we slam a docstring on here?

Reply:
lolol I wanted to make it _more_ private but yeah it's redundant. will add docstrings.

Reply:
could do a dunder to make it more private too https://docs.python.org/3/tutorial/classes.html#private-variables